test/system_settings_test.rb

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SystemSettings"
+
+describe "Yast::SystemSettings" do
+  Yast.import "Bootloader"
```

Comment:
Just a "mental" reason (I tend to put there imports I use in mocks). But it does not make sense :) So I'll move it.